packages/app-content-pages/server/server.js

```diff
@@ -22,3 +27,18 @@ app.prepare().then(() => {
     console.log(`> Ready on http://localhost:${port}`)
   })
 })
+
+function setCacheHeaders (req, res) {
+  let maxAge = 60 // 1 minute
+
+  if (isJsRequest(req)) {
+    maxAge = 31536000 // 1 year
+  }
+
+  res.setHeader('Cache-Control', `max-age=${maxAge}`)
+}
+
+function isJsRequest(req) {
+  const regex = /\.(?:js)$/i
```

Comment:
The regex works, any reason for the non capture group? Could this be simplified to `/\.js$/i`? I'm probably missing something important here. 

Super minor - just thinking about this getting copied around / becoming the way it to check file extensions.

Reply:
No, that could absolutely be simplified to that